ipcsdk/ipcaddress.go

```diff
@@ -0,0 +1,34 @@
+package ipcsdk
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/filecoin-project/go-address"
+)
+
+// UndefIPCAddress creates a convenient type to define
+// undefined IPC address
+var UndefIPCAddress = IPCAddress{}
+
+// IPCAddress adds subnet information to raw Filecoin addresses
+type IPCAddress struct {
```

Comment:
Should we rename it to Address? In that case, it would be `ipcsdk.Address` not `ipcsdk.IPCAddress`? 
If the package was named `sdk` then `sdk.IPCAddress` would work.

Reply:
I would leave it as `IPCAddress` as there is already `address.Address` in the code.

Reply:
Will stick with `sdk.IPCAddress` incorporating both feedbacks. Thanks!